src/main/resources/hudson/plugins/sshslaves/verifiers/MissingVerificationStrategyAdministrativeMonitor/message.jelly

```diff
@@ -22,8 +22,18 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
-    <div class="warning">
-        <p>SSH Host Key Verifiers are not configured for all SSH slaves on this Jenkins instance. This could leave these slaves open to man-in-the-middle attacks. <a href="${rootURL}/computer/">Update your slave configuration</a> to resolve this.</p>
-    </div>
+<j:jelly xmlns:j="jelly:core">
+
+<j:if test="${!it.isTheNewDesignAvailable}">
```

Comment:
Should just be made into a Groovy view so there's no change to plugin classes needed.

Reply:
@daniel-beck Agree although I'm not fan of Groovy for that purpose. I wanted to offer to two different examples. This one, and this https://github.com/jenkinsci/github-plugin/pull/179, where the maintainers are already using Groovy.

Reply:
I am fine with a non-Groovy implementation. As we discussed with @recena in the chat, we rather need a macro or tag, which would check the Jenkins core version

Reply:
@oleg-nenashev Yes, a Jelly tag would be helpful. I'll find some of time for it.

Reply:
could use j:choose + j:when + j:otherwise
And should use a var to avoid code duplication

Reply:
@ndeloof A `var`for what?

Reply:
Remember, we are considering two cases just temporarily. When the baseline is updated, we don't need it.

Reply:
a var to define the css class to be used.
```xml
<j:choose>
  <j:when test="${it.isTheNewDesignAvailable}">
     <j:set var="css" value="warning">
  </j:when>
  <j:otherwise>
     <j:set var="css" value="alert alert-warning">
  </j:otherwise>
</j:choose>
<div class="${css}">
...
```

makes me wonder, but it seems one could even use this simpler form:
```xml
<div class="${it.isTheNewDesignAvailable() ? 'alert alert-warning' : 'warning'}">
...
```

by the way, using `isNewDesignAvailable()` as method name would allow to use property style notation `${it.isNewDesignAvailable}` (adding "The" sounds weird to me).

I'm pretty sure this has been intensively investigated, but can't the CSS just target "warning" class efficiently enough within an administrative monitor context so all this isn't required ? I would understand this is required it he DOM structure was different, but here this sounds to only be a question of CSS alias.

Reply:
It seems to be you are not taking into account the paragraph `<p>`, needed in the first case.

By the way, as you can read in the article, this is temporal. It could be removed when the baseline is updated.

Reply:
And by the way, this is the simplest case where the content is more or less similar (but not identical). However, you will find more complex cases here https://github.com/jenkinsci/jenkins/pull/2857

I don't see the value of using a `var`.